### DISCUSSION TITLE
time zone

### DISCUSSION THREAD

**Discussion by NeloReis** [Budibase/budibase#2862] [Help] [answered]
I'm getting discrepancies in the Date Fields .. how do I set up the time zone? so the date interpretation (EX Repeater) comes out right.
as is stands  (time are a bit erratic) but if I try to read a Date field from Data Source the repeater sends a date with an additional  day ( wrong time zone) 
is there a way to define the correct time zone? on docker? DB config?
Thanks

**Comment by mjashanks** [comment, marked as answer]
Hey @NeloReis 

Budibase doesn't save timezones. We save the data "as is", i.e. the exact data sent by the app/browser. So, if I send the `Now` date, which is `2021-10-06 20:28` - it gets saved as that, and loaded as that, regardless of the viewer's timezone.

I have just been switching my timezone and verified that this is the behaviour.

What would you expect / like to be able to do with timezones?

**Reply by NeloReis** [reply]
Hi, thank you for your explanation.  i was having issues with the date aparantly because ( on the helper secction) it states (date now)
and someone told me that the correct form is just NOW. 
I'm on an intricate part of math development and having loads o fun.
As soon as i go back to dealing with inputs and dates .. ill do extensive tests.. and let you know. but thank you so much.
All your info has been extremely helpful.
any news on that JS for math you told me about? is this year or  next?.. just a ball park ..

**Reply by mjashanks** [reply]
You're welcome!

JS - end of October 2021!

**Reply by NeloReis** [reply]
awesome news.. :D
